pom.xml

```diff
@@ -188,21 +177,27 @@
     <!-- ======================================================================= -->
     <dependencies>
         <dependency>
-            <groupId>org.owasp.antisamy</groupId>
-            <artifactId>antisamy</artifactId>
-            <version>1.6.4</version>
+            <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
+            <artifactId>owasp-java-html-sanitizer</artifactId>
+            <version>20211018.2</version>
```

Comment:
why not last version `20220608.1`?

Reply:
Fixed in 8d3b9fbc1e3feab8d591e0e480685b8d36ed8f47